navbar-service.js

```diff
@@ -28,6 +32,23 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  service.site = {
+    title: config.data.siteTitle,
+    brand: config.data.style.brand,
+    url: '/'
+  };
```

Comment:
It was my expectation that we would be setting the tenant value in a route resolver.  Something along these lines:
```
    .when('/demo/:tenant/landing/', {
      vars: {
        title: 'Landing Page',
        navbar: {display: 'tenant'}
      },
      template: '<demo-landing-page></demo-landing-page>',
      resolve: {
        navbar: function($route) {
          var params = $route.current.params;
          return tenantService.get(params.tenant)
            .then(function(tenant) {
              brNavbarService.tenant = tenant.navbar;
            });
        }
      }
    })
```